### PR TITLE
refactor: move to client load

### PR DESCRIPTION
**Pull request by MaiaS**
## WHAT IT DOES:

move readability initial load to client side load, address bug with reload url

### CODE REVIEW

File: src/utils/generateLinkToShare.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 
-export function encodeStateToUrl(originalContent: string, responseObject: ResponseType): string {
+export function encodeStateToUrl(originalContent: string, responseObject: ResponseType, songDetails?: string): string {
   const baseUrl = process.env.BASE_URL || window.location.origin;
-  const queryString = `?result=${encodeURIComponent(JSON.stringify(responseObject))}&type=${encodeURIComponent(
-    responseObject.type,
-  )}&original=${encodeURIComponent(originalContent)}`;
+  const queryString = `?result=${encodeURIComponent(JSON.stringify(responseObject))}&original=${encodeURIComponent(
+    originalContent,
+  )}${songDetails ? `&songDetails=${encodeURIComponent(songDetails)}` : ""}`;
```
Comment (pratikmdhr) [inline src/utils/generateLinkToShare.ts:7]: ✔️

File: src/components/Result/ResultContent/SongResult.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ const SongResult = ({
           <div className="hidden md:block">
             <ShareLinkButton
               trackShare={trackShare}
+              songDetails={songDetails}
```
Comment (pratikmdhr) [inline src/components/Result/ResultContent/SongResult.tsx:36]: you missed adding `songDetails` to the `<ShareLinkButton/>` that appears on the mobile layout (Line 46 of the same file)

**Review by pratikmdhr — CHANGES_REQUESTED** [review]
its looks all good! 🙏
one small fix and its good to merge!

File: src/components/Result/ResultContent/SongResult.tsx (hunk 2 above)
Comment (MaiaS) [reply src/components/Result/ResultContent/SongResult.tsx:36]: ah nice catch

**Review by pratikmdhr — APPROVED** [review]
✅ thanks for fixing it!

**Pull request merged**